src/Command/Api/ApiListCommand.php

```diff
@@ -10,34 +10,33 @@
 /**
  *
  */
-class ApiListCommand extends CommandBase {
+class ApiListCommand extends ApiListCommandBase {
 
   protected static $defaultName = 'api:list';
 
   /**
    * {inheritdoc}.
    */
   protected function configure() {
-    $this->setDescription("<fg=cyan>There are more hidden API commands! Run api:list to see them all.</>")
+    $this->setDescription("List all API commands")
```

Comment:
Do we not need to keep the color here to have it stand out?

Reply:
I don't think so. See uploaded image.

Reply:
I saw, thus why I was asking.

Reply:
I feel that the api:list command is less important now that we have a number of list commands. So, it doesn't seem necessary for it to stand out. I think we'd prefer the user to run one of the commands with less output.